test-it/karma/helloworld/HelloWorld.Test.js

```diff
@@ -28,9 +28,26 @@ describe('Vireo loaded as a global in the browser', function () {
         expect(result).toBe('Hello, sky. I can fly.\n');
     });
 
-    it('can run HelloWorld Async', function (done) {
-        var Vireo = window.NationalInstruments.Vireo.Vireo;
-        var vireo = new Vireo();
+    it('can run HelloWorld async using a callback', async function () {
```

Comment:
Cannot? Or perhaps test says that now it throws?

Reply:
good catch! updated the test name